### ISSUE TITLE
Conda found conflicts

### ISSUE THREAD

**Issue by fuesseler** [oschakoory/RiboTaxa#4]
Hello,
I ran into some installation issues while following the README instructions: installing  the conda environments fails due to conflicts. 

After Running:
`bash conda_virt_env.sh`

This error appears:
`
Creating virtual environment and installing RiboTaxa...  Thu Apr 13 14:17:28 CEST 2023
Collecting package metadata (repodata.json): done
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed`

and then after a long time, a massive list of conflicts is listed.
Can you check if there is something wrong with the .yml files?

Thanks,
Flora

**Comment by oschakoory** [comment]
Hi,
I have checked/tested the installation and the yml files, i found no conflicts.

Can you send me the output of your conda environment, to see with which environment you are having conflicts?

```bash
conda info --envs
```

Please ensure that you have the latest version of miniconda (23.3.1) installed before installing RiboTaxa. Update conda if need be, before re-installing RiboTaxa:

```bash
conda update --all --yes
```

Let me know if you still have conflicts.

Oshma

**Comment by fuesseler** [comment]
Hi, 
Thanks for getting back so quickly! I attached the very long conflicts output for the first environment (RiboTaxa_py27) below.
[ribotaxa_conda_conflicts.txt](https://github.com/oschakoory/RiboTaxa/files/11250595/ribotaxa_conda_conflicts.txt)

Actually, on my local PC the conda installation worked just fine, so it must be some problem with the HPC environment / my miniconda3 version on there. The problem persisted even after updating to the newest version of miniconda. I had the same problem when installing another pipeline via.yml file, so it must be on my end.

I solved the issue by installing on local PC then using conda-pack and uploading the tar.gz file of the environment to my miniconda3 directory on the HPC. It's running now, so fingers crossed that it'll finish without problems.

Thanks for the help!
Flora
